api/src/infrastructure/kubernetes/payloads.rs

```diff
@@ -179,13 +178,13 @@ pub fn deployment_payload(
             .collect::<Vec<_>>()
     });
 
-    let volumes = strategy.volumes().map(|volumes| {
-        let volumes = volumes
+    let files = strategy.files().map(|files| {
```

Comment:
See comment above: use K8s primitive.

```suggestion
    let volumes = strategy.files().map(|files| {
```